src/cljs/doit/views.cljs

```diff
@@ -0,0 +1,43 @@
+(ns doit.views
+  (:require [re-frame.core :as rf]
+            [reagent.core :as reagent]
+            [doit.subs :as subs]
+            [doit.events :as events]))
+
+(defn header []
+  [:div.header
+   [:h1 "DO•IT"]])
+
+(defn add-todo-form []
```

Comment:
I haven't used re-frame in the past. But I'm assuming you can write tests for `add-todo-form` and `todos-panel` by testing the event handlers and asserting on the view state.

Refer to this maybe: https://github.com/Day8/re-frame/blob/master/docs/Testing.md